docs/source/demo.rst

```diff
@@ -7,41 +7,63 @@ Demo
 Running a demo site
 ===================
 
-Atramhasis comes with a demo site include. This allows you to quickly evaluate
-and inspect the software. To get started, just download Atramhasis from pypi and
-install it. We recommend doing this in a virtualenvironment.
+Checking a working instance of the Atramhasis can be done at `the Flanders
+Heritage Thesaurus <https://thesaurus.onroerenderfgoed.be>`_ or by running a
+demo yourself. This allows you to quickly evaluate and inspect the software. 
+This can be done through the `cookiecutter` package. 
 
 .. code-block:: bash
 
     $ mkvirtualenv atramhasis_demo
-    $ pip install -U atramhasis
+    $ pip install -U cookiecutter
+
+Once cookiecutter is installed, you use it to generate the demo site.
+
+.. code-block:: bash
 
+    $ cookiecutter gh:OnroerendErfgoed/atramhasis_demo_cookiecutter
 
-Once Atramhasis is installed, you can call upon a pyramid scaffold to generate
-the demo site.
+Running this command will ask a few questions. Just accept the default answers,
+unless you want to give your project a different name. After the
+cookiecutter command, there should be a directory with the name of your
+project (default: atramhasis_demo). Enter this directory and install
+requirements:
 
 .. code-block:: bash
 
-    $ pcreate -s atramhasis_demo atramhasis_demo
     $ cd atramhasis_demo
+    $ pip install -r requirements-dev.txt
+    $ pip install -e .
 
-This creates a local demo package you can run with just a few more commands:
+Now it's time to setup our database (a simple SQLite database) and add some 
+testdata:
 
 .. code-block:: bash
 
-    # setup
-    $ pip install -r requirements-dev.txt
-    $ python setup.py develop
-    # download and install client side libraries
-    $ cd atramhasis_demo/static
-    $ npm install
-    $ cd ../..
-    # create or upgrade database
     $ alembic upgrade head
-    # intialize sample data
+    # fill the database with data
     $ initialize_atramhasis_db development.ini
-    # generate full RDF dumps (not necessary for basic functionality)
+
+Optionally, we can create RDF dumps, but this is not necessary for basic
+functionality:
+
+.. code-block:: bash
+
     $ dump_rdf development.ini
+
+Almost done! All we need now are some frontend dependencies:
+
+.. code-block:: bash
+
+    $ cd atramhasis_demo/static
```

Comment:
we assume that the user accepted the default repository name. We might use something like `cd <package>/static`.

Reply:
True. But this has the advantage that you can just copy paste the instructions, without having to think about them. So I'd leave it as it is. But we could update the doc intro a bit more, something like "If you do not accept the default name atramhasis_demo, please update the name atramhasis_demo in the instructions below."